backend/vecsim_app/entrypoint.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
+set -eux
```

Comment:
abort if any command fails, print all commands to be executed, disallow unset vars